ontopy/ontology.py

```diff
@@ -200,10 +200,12 @@ def __dir__(self):
         lst = list(self.get_entities(imported=self._dir_imported))
         if self._dir_preflabel:
             dirset.update(
-                _.prefLabel.first() for _ in lst if hasattr(_, "prefLabel")
+                _.prefLabel.en.first() for _ in lst if hasattr(_, "prefLabel")
```

Comment:
This only works for preflabels that are explicitly defined to be English. I think it would be better to allow `dirset` to contain both normal strings and loc-strings.

Another note, the name `_` should only be used for unused variables. Both for semantic reasons and because it is overwritten by the standard Python interpreter.